ui.frontend/src/view/FormFieldBase.js

```diff
@@ -608,16 +627,10 @@ class FormFieldBase extends FormField {
      */
     updateDescription(descriptionText) {
         if (typeof descriptionText !== 'undefined') {
-            const sanitizedDescriptionText = window.DOMPurify ?  window.DOMPurify.sanitize(descriptionText) : descriptionText;
+            let sanitizedDescriptionText = window.DOMPurify ?  window.DOMPurify.sanitize(descriptionText) : descriptionText;
             let descriptionElement = this.getDescription();
             if (descriptionElement) {
-                let pElement = descriptionElement.querySelector("p");
-                if (!pElement) {
-                    // If the description is updated via rule then it might not have <p> tags
-                    pElement = document.createElement('p');
-                    descriptionElement.appendChild(pElement);
-                }
-                pElement.textContent = sanitizedDescriptionText;
+                descriptionElement.innerHTML = sanitizedDescriptionText;
```

Comment:
Can we optimize this ? if the content is already the same, let's not update the HTML. This would be a NO-OP operation in few cases which needs to be handled